src/domains/idpay-app/api/idpay_onboarding_workflow/openapi.onboarding.yml.tpl

```diff
@@ -0,0 +1,378 @@
+        '404':
+          description: The requested ID was not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDto'
+              example:
+                code: 0
+                message: string
+        '429':
+          description: Too many Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDto'
+              example:
+                code: 0
+                message: string
+        '500':
+          description: Server ERROR
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDto'
+              example:
+                code: 0
+                message: string
+  /consent:
+    put:
+      tags:
+        - onboarding
+      summary: Save the consensus of both PDND and self-declaration
+      operationId: consentOnboarding
+      requestBody:
+        description: 'Unique identifier of the subscribed initiative, flag for PDND acceptation and the list of accepted self-declared criteria'
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ConsentPutDTO'
+            example:
+              initiativeId: string
+              pdndAccept: true
+              selfDeclarationList:
+                - code: string
+                  accepted: true
+      responses:
+        '202':
+          description: Accepted - Request Taken Over
+          content:
+            application/json: { }
+        '400':
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDto'
+              example:
+                code: 0
+                message: string
+        '401':
+          description: Authentication failed
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDto'
+              example:
+                code: 0
+                message: string
+        '404':
+          description: The requested ID was not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDto'
+              example:
+                code: 0
+                message: string
+        '429':
+          description: Too many Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDto'
+              example:
+                code: 0
+                message: string
+        '500':
+          description: Server ERROR
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorDto'
+              example:
+                code: 0
+                message: string
+  '/{initiativeId}/status':
```

Comment:
How can the path start with the {initiativeId}? And why we need the `status` suffix?

Why not a simple GET on `/initiative/{initiativeId}` to get the status? (also here returns the possible requirement to change name to the resource, "initiative" seem not appropriate)